src/app/model/music.model.ts

```diff
@@ -0,0 +1,10 @@
+import { Album } from './album';
+import { Artist } from './artist.model';
+
+export interface Music {
```

Comment:
I was thinking in an interface named _**Song**_ but this works anyways.

Reply:
changed

Reply:
where is the change?

Reply:
you are crazy